filters/ThirdParty/filter_238_PolskiFiltrAdg/metadata.json

```diff
@@ -9,8 +9,8 @@
   "groupId": 7,
```

Comment:
Fix the filter description as well, mention that it's now included into the main subscription.

@AdamWr could you please assist?